projects/extension/sql/idempotent/900-semantic-catalog-init.sql

```diff
@@ -3,12 +3,12 @@
 -------------------------------------------------------------------------------
 -- initialize_semantic_catalog
 create or replace function ai.initialize_semantic_catalog
-( "name" pg_catalog.name default 'default'
-, embedding pg_catalog.jsonb default null
+( embedding pg_catalog.jsonb default null
 , indexing pg_catalog.jsonb default ai.indexing_default()
 , scheduling pg_catalog.jsonb default ai.scheduling_default()
 , processing pg_catalog.jsonb default ai.processing_default()
 , grant_to pg_catalog.name[] default ai.grant_to()
+, "name" pg_catalog.name default 'default'
```

Comment:
nit: When doing name=>'blah' do you have to quote name? If so? consider changing to 'catalog_name' to avoid the reserved keyword?

Reply:
actually catalog_name is used elsewhere so maybe good to switch anyway?

Reply:
https://github.com/timescale/pgai/pull/289/commits/12b058dd2c3b9fb716614da04dd385833248f3d1